src/main/java/org/xstefank/webhook/WebHookEndpoint.java

```diff
@@ -7,21 +7,26 @@
 import org.xstefank.api.GitHubAPI;
 import org.xstefank.check.SkipCheck;
 import org.xstefank.check.TemplateChecker;
+import org.xstefank.verification.BadFormatException;
+import org.xstefank.verification.VerificationHandler;
 import org.xstefank.model.CommitStatus;
 import org.xstefank.model.Utils;
 import org.xstefank.model.yaml.FormatConfig;
+
 import javax.ws.rs.Consumes;
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.core.MediaType;
 import java.io.File;
 import java.io.IOException;
+
 import static org.xstefank.check.TemplateChecker.TEMPLATE_FORMAT_FILE;
 
 @Path("/")
 public class WebHookEndpoint {
 
     private static final Logger log = Logger.getLogger(WebHookEndpoint.class);
+    private static VerificationHandler verificationHandler = new VerificationHandler();
```

Comment:
I don't think it's required to keep this handler in a field as its only used in one method, which is (I may be wrong here) called only once on the following line